frontend/i18n/pl/projects/overview.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  createProjectTitle: 'Utwórz projekt',
+  projectName: 'Nazwa projektu',
+  projectType: 'Typ projektu',
+  textClassification: 'Klasyfikacja tekstu',
+  sequenceLabeling: 'Etykietowanie sekwencji',
+  sequenceToSequence: 'Sekwencja do sekwencji',
+  randomizeDocOrder: 'Losuj kolejność dokumentów',
+  shareAnnotations: 'Udostępnij adnotacje wszystkim użytkownikom',
+  remove: 'Usuń',
+  deleteProjectTitle: 'Usuń projekt',
+  deleteProjectMessage: 'Czy na pewno chcesz usunąć te projekty?',
+  search: 'Szukaj',
+  projectTypes: [
+    'Klasyfikacja tekstu',
+    'Etykietowanie Sekwencji',
+    'Sekwencja do sekwencji',
+    'Wykrywanie zamiarów i wypełnianie szczelin',
+    'Klasyfikacja obrazu',
+    'Mowa na SMS',
```

Comment:
I think instead of `Mowa na SMS` it is better to write `Mowa na tekst` or `Głos na tekst` XD